make/modules/jdk.javadoc/Gendata.gmk

```diff
@@ -72,7 +72,7 @@ $(JDK_JAVADOC_DIR)/_element_lists.marker: \
     $(MODULE_INFOS)
 	$(call MakeTargetDir)
 	$(call LogInfo, Creating javadoc element lists)
-	$(RM) -r $(ELEMENT_LISTS_DIR)
+	$(RM) $(ELEMENT_LISTS_DIR)/element-list-{$(call CommaList,$(call sequence,$(GENERATE_SYMBOLS_FROM_JDK_VERSION),$(JDK_SOURCE_TARGET_VERSION)))}.txt
```

Comment:
Good to see that it worked! I would only wish that you found a way to break up the line. Long lines make future side-by-side reviews and 3-way merges hard. We don't enforce strict 80, but try to stay in some reasonable ballpark in the build files.

I think both CommaList and sequence are ok with whitespace in their parameters. Otherwise you could pre-calculate the numbers list in a variable before the recipe.

Reply:
yep, i'll have a look to see what I can do, not helped with existing variables being so long!! (GENERATE_SYMBOLS_FROM_JDK_VERSION)

Reply:
yep, split on CommaList seems better and works fine, see revision

Reply:
Does it work if you have space after the comma in the call to sequence? We try to have a whitespace after comma where possible (and try to make function handle that case). If it does not work, we should really fix sequence...